src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -68,6 +56,88 @@ MenuGame::MenuGame(Steam::steam_manager& mSteamManager,
                 downAction();
             }
         };
+    game.onEvent(Event::EventType::KeyReleased) +=
+        [this](const Event& mEvent)
+        {
+            if(!noActions) return;
```

Comment:
The name `noActions` is not immediately clear -- please rename the variable or add a comment. I'm not sure what it means

Reply:
I named it after the same variable that is part of the DialogBox PR.
When 1 or higher it's a signal to MenuGame that all actions like upAction(), downAction() etc. are temporarely disabled since user should be able to assign a key to a bind without any of those actions be triggered in the process, and the key presses can be used by the function below to be assigned to the binds.

If it's not clearn enough I will change the name and also change the name in the other PR for consistency.